AI-and-Analytics/End-to-end-Workloads/LidarObjectDetection-PointPillars/README.md

```diff
@@ -0,0 +1,132 @@
+# `PointPillars` Sample
+This sample performs 3D object detection and classification using data (point cloud) from a LIDAR sensor as input. The Intel® oneAPI implementation is based on the paper 'PointPillars: Fast Encoders for Object Detection from Point Clouds' [1] and the implementation in [2]. It shows how Intel® oneAPI kernels (using SYCL and DPCPP) can be used in combination with the Intel® Distribution of OpenVINO™ toolkit for CNN inference.
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04
+| Hardware                          | Skylake with GEN9 or newer / Intel Xe Graphics
+| Software                          | Intel® oneAPI DPC++/C++ Compiler, Intel® Distribution of OpenVINO™ toolkit
+| What you will learn               | How to combine Intel® Distribution of OpenVINO™ toolkit and Intel® oneAPI to offload the computation of a complex workload to one of Intel's supported accelerators (e.g. GPU or CPU)
+| Time to complete                  | 30 minutes
+
+## Purpose
+PointPillars is an AI algorithm, that uses LIDAR point clouds to detect and classify 3D objects in the sensor environment. For this purpose, the algorithm consists of 5 main steps. First a pre-processing of the point cloud is performed. This is realized with the help of kernels implemented in oneAPI. Afterward, the preprocessed data is used by a so-called Pillar Feature Extraction (PFE) CNN to create a 2D image-like representation of the sensor environment. For the inference, this sample uses the Intel® Distribution of OpenVINO™ toolkit. It follows another oneAPI processing step before a second CNN inference for the so-called Region Proposal Network (RPN) is executed using the OpenVINO™ toolkit. Finally, the output data (object list) is post-processed and filtered, which is again performed in oneAPI kernels. An overview of the data flow is given in image below:
+![Overview](data/point_pillars_overview.png)
+
+
+By default the application will use 'host' as the Intel® oneAPI execution device and CPU for Intel® Distribution of OpenVINO™ toolkit inferencing part. The Intel® oneAPI execution device and inferencing device are displayed in the output along with elapsed time of each of the five steps described above. For more details refer to section: [Execution Options for the Sample Program](#execution-options-for-the-sample-program).
```

Comment:
his section refers to the 5 steps above... reading through it, it's hard to pull 5 steps out (I get 4). So people unfamiliar with the technology may struggle to comprehend. Move the steps into an organized list format to enhance clarity. Please note I made some edits to your GitHub version of the readme for grammar and punctuation.

Reply:
Hi Joe,
I updated the readme to provide this in a list form and aligned it with our illustration of the algorithm (6 steps).

Reply:
That looks great thank you